projects/find/find.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+
+"""This method is created to find all files and directories, given a specific path. Optionally, your can use -name flag
+to search only the files that match to a specific name"""
+
+
+def find(path, option=None, filename=None):
+    """this method uses a bfs approach to resolve to the problem of finding all files and directories."""
+    if not os.path.exists(path):
+        return "Path doesn't exist or wrong: please try again"
+    queue = []
+    queue.append(path)
```

Comment:
instead of creating the list and then appending, you can create the list with `path` already there: `queue = [path]`